detox/index.d.ts

```diff
@@ -1030,29 +1030,29 @@ declare global {
         interface WebExpect<R = Promise<void>> {
             /**
              * Negate the expectation.
-             * @example await expect(webview.element(by.htmlId('UniqueId205'))).not.toExist();
+             * @example await expect(webview.element(by.web.id('UniqueId205'))).not.toExist();
              */
             not: this;
 
             /**
              * Expect the element content to have the `text` supplied
              * @param text expected to be on the element content
              * @example
-             * await expect(webview.element(by.htmlId('UniqueId205'))).toHaveText('ExactText');
+             * await expect(webview.element(by.web.id('UniqueId205'))).toHaveText('ExactText');
```

Comment:
`webview.element` should be changed also to `web.element` right ?

Reply:
not necessarily, if you say `const webview = web(by.id('myWebView));` 
but to avoid confusion — yeah, better to do.